### PR TITLE
refactor/docs(SQS): endpoints

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #XXX

## What is the purpose of the change

This PR modifies the endpoints to have prefixes by resource. Additionally, it documents every endpoint for future integrators.

## Testing and Verifying

Tested locally.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: ingest/sqs/README.md [hunk 1]
```diff
@@ -11,6 +11,276 @@ The use case for this is performing certain data and computationally intensive t
...
+      ],
+      "out_amount": "1803",
+      "in_amount": "1000000"
+    }
+  ],
+  "effective_fee": "0.006000000000000000"
+}
+```
+
+1. POST `/router/store-state`
```
Comment (p0mvn) [inline ingest/sqs/README.md:261]:
```suggestion
5. POST `/router/store-state`
```

File: ingest/sqs/README.md [hunk 2]
```diff
@@ -11,6 +11,276 @@ The use case for this is performing certain data and computationally intensive t
...
+
+Parameters:
+- `amountIn` integer amount of tokenIn
+- `denomIn` string denom of tokenIn
+- `denomOut` string denom of tokenOut
+
+Response example:
+
+```bash
+curl "https://sqs.osmosis.zone/router/quote?tokenIn=1000000uosmo&tokenOutDenom=uion" | jq .
```
Comment (jonator) [inline ingest/sqs/README.md:82]: This curl doesn't appear to match the parameters above. Also description doesn't match params

Comment (p0mvn) [reply ingest/sqs/README.md:82]:
```
/router/quote?tokenIn=<amount><denomIn>&tokenOutDenom=<denomOut>
```

```
https://sqs.osmosis.zone/router/quote?tokenIn=1000000uosmo&tokenOutDenom=uion
```

- `amountIn` integer amount of tokenIn
- `denomIn` string denom of tokenIn
- `denomOut` string denom of tokenOut

where:
- `amountIn` is 1000000
- `denomIn` is uosmo
- `denomOut` is uosmo

Comment (p0mvn) [reply ingest/sqs/README.md:82]: Noticed related issues and fixed here:
[e7e48d1](https://github.com/osmosis-labs/osmosis/pull/7026/commits/e7e48d12d02409f6192daff73d268d40b75f5342)

Comment (p0mvn) [reply ingest/sqs/README.md:81]: I see the confusion, please check this update:
[fcb6cec](https://github.com/osmosis-labs/osmosis/pull/7026/commits/fcb6cecaf924d3a228afcc457c36ad267e0373b9)

Comment (jonator) [reply ingest/sqs/README.md:81]: OK that's what I had before

Comment (jonator) [reply ingest/sqs/README.md:81]: OK that's what I had before

**Review by jonator — APPROVED** [review]

**Pull request merged**